distribution/docs/src/main/resources/_contents/_core-concepts/metadata-intro-contents.adoc

```diff
@@ -0,0 +1,56 @@
+
+== Understanding Metadata and Metacards
+
+Metadata is information about a resource, organized into a schema to make it possible to search against.
+The ${ddf-catalog} stores this metadata and allows access to it.
+Metacards are single instances of metadata, representing a single record, in the Metadata Catalog (MDC).
+Metacards follow one of several schemas to ensure reliable, accurate, and complete metadata.
+Essentially, Metacards function as containers of metadata.
+
+=== Populating Metacards During Ingest
+
+Upon ingest, a metacard transformer will read the data from the ingested file and populate the fields of the metacard.
+Exactly how this is accomplished depends on the origin of the data, but most fields (except id) are imported directly.
+
+=== Searching Metadata
+
+${branding} provides the capability to search the Metadata Catalog (MDC) for metadata.
+There are a number of different types of searches that can be performed on the MDC, and these searches are accessed using one of several interfaces.
+This section provides a very high level overview of introductory concepts of searching with ${branding}.
+These concepts are expanded upon in later sections.
+
+==== Search Types
+
+There are four basic types of metadata search.
+Additionally, any of the types can be combined to create a compound search.
+
+===== Contextual Search
+
+A contextual search is used when searching for textual information.
```

Comment:
Might consider mentioning that this searches *all* textual metadata fields by default, and the distinction between that and a contextual search on a specific attribute. Not sure if that's too detailed for this section.